pkg/registry/rbac/validation/rule.go

```diff
@@ -176,11 +177,14 @@ func (d *roleBindingDescriber) String() string {
 }
 
 func (r *DefaultRuleResolver) VisitRulesFor(user user.Info, namespace string, visitor func(source fmt.Stringer, rule *rbacv1.PolicyRule, err error) bool) {
+
+	userTenant := user.GetTenant()
+
 	if clusterRoleBindings, err := r.clusterRoleBindingLister.ListClusterRoleBindings(); err != nil {
 		if !visitor(nil, nil, err) {
 			return
 		}
-	} else {
+	} else if userTenant == "system" {
```

Comment:
Minor: we have a const definition, metav1.TenantSystem = "system"

Also, we have metav1.TenantDefault = "default"

Reply:
fixed